### ISSUE TITLE
Breakpoint markers don't show properly

### ISSUE THREAD

**Issue by roangel** [emacs-lsp/dap-mode#374]
I am new to using dap-mode. I am testing it now with python, and this is how I visually see breakpoints when adding them:
Inactive:
![image](https://user-images.githubusercontent.com/21340299/94370910-0a03b480-00f3-11eb-9aad-fa131680a873.png)
Active:
![image](https://user-images.githubusercontent.com/21340299/94370913-0d973b80-00f3-11eb-95c3-2772c5de074e.png)

Is it normal that they are super small? Can I make them a bit bigger somehow in the configuration? This is my current config:

```
(setq dap-python-executable "python3")
;; Add languages
(require 'dap-ui)
(require 'dap-python)

(dap-auto-configure-mode 1)

(provide 'my_dap_mode)
```

**Comment by ericdallo** [comment]
This is an issue with the size of your fringe(the left space where shows the breakpoint), I think you can increase it setting something like:
```elisp
(setq left-fringe-width 16)
```

**Comment by roangel** [comment]
Aha, ok, thanks, it was solved by changing

`(set-fringe-style (quote (2 . 8)))` to
`(set-fringe-style (quote (12 . 8)))`

**Comment by yyoncho** [comment]
@Awannaphasch2016 I am not sure what the solution here is, I have to consult with emacs-devel. For now, as a workarround you may change dap-ui-pending-breakpoint-face and dap-ui-verified-breakpoint-face.

**Comment by avucic** [comment]
When I enter dap-mode git status symbols disappear. Also no dap breakpoints.

After I eval `(set-window-buffer nil (current-buffer))`  I can see again markers (including git status). 
Otherwise, I have to close and open the buffer again to be able to see thouse markers. 

Also with the new doom installation, I can't see git status markers by default. I have this in my config`(fringe-mode nil)` to get them.
Not sure if this somehow helps or makes sense (I'm new to using doom).

**Comment by dfpetrin** [comment]
Same as @avucic , when I enter dap-mode all of the fringe indicators disappear. Breakpoints, current line marker, and git-gutter status. Evaluating `(set-window-buffer nil (current-buffer))` brings them back. I'm trying to figure out if there is a hook I can add that to as a workaround...

**Comment by dfpetrin** [comment]
Ok if you're using Doom emacs then adding this to your `config.el` should work:
```
(add-hook! +dap-running-session-mode
  (set-window-buffer nil (current-buffer)))
```

Definitely a hacky workaround, but seems to be working for me.

**Comment by child404** [comment]
Hi @dfpetrin @avucic ! Have you fixed that issue another way, without the hook? I'm also experiencing this issue. Unfortunately, the workaround with hook provided by @dfpetrin didn't work for me. So I added hook to dap-stack-frame-changes-hook, but it worked only when I enter dap. Also, seems that the issue appears when the stack frame (i.e. all dap panes) appears/disappears

**Comment by eeowaa** [comment]
This appears to be a problem caused by the `doom-themes-ext-treemacs` library. The following code worked for me:

``` elisp
(after! doom-themes-ext-treemacs
  (with-eval-after-load 'treemacs
    (remove-hook 'treemacs-mode-hook #'doom-themes-hide-fringes-maybe)
    (advice-remove #'treemacs-select-window #'doom-themes-hide-fringes-maybe)))
```

If you happen to be using Doom Themes outside of Doom Emacs, try replacing `after!` with `eval-after-load`.

@child404 Does my solution work for you, as well?

**Comment by nsulmol** [comment]
@eeowaa I can confirm that your fix works!

For reference: when I first tried it, it did not resolve the issue. However, this was because my lsp-treemacs had a byte compilation issue (it had obtained lsp-treemacs compiled files [.elc] that were a different version from my treemacs version). This could be seen in the 'messages' buffer, where there was a "Symbol's function definition is void" message tied to lsp-treemacs.

Once I resolved this issue (by deleting the associated .elc files and re-running emacs), your fix worked.

Thank you!
